simulation/costs.py

```diff
@@ -331,3 +331,43 @@ def estimate_boiler_upgrade_scheme_grant(
 
     if heating_system == HeatingSystem.HEAT_PUMP_GROUND_SOURCE:
         return 7_500
+
+
+def estimate_extended_boiler_upgrade_scheme_grant(
```

Comment:
Is there a test for this function?

Reply:
I have now added tests for this function.